3741/3/helpers/user_helper.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+
+module UserHelper
```

Comment:
Methods described here is just a private methods of controller

Reply:
yes and it was initially so, but i got an reek error **InstanceVariableAssumption**, and replace methods in that helper

Reply:
So you should solve it instead of escaping from it

Reply:
done